src/main/java/com/socialbox/repository/UserRepository.java

```diff
@@ -1,8 +1,16 @@
 package com.socialbox.repository;
 
 import com.socialbox.model.User;
+import java.util.List;
+import java.util.Optional;
 import org.springframework.data.mongodb.repository.MongoRepository;
 
-public interface UserRepository extends MongoRepository<User, String> {
```

Comment:
We cannot have custom functions with MongoRepository. It is only useful for simple CRUD operations. For other use, we need to use `MongoTemplate`